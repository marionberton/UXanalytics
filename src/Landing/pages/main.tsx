import React from 'react';

import { Home } from '../components/sections/home/home';
import { Demo } from '../components/sections/demo/demo';
import { Benefits } from '../components/sections/benefits/benefits';
import { Cta } from '../components/sections/cta/cta';

export const Main: React.FC = () => {
  return (
    <>
      <Home />
      <Demo />
      <Benefits />
      <Cta />
    </>
  );
};
