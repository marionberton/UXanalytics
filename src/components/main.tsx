import React from 'react';

import { Home } from './sections/home/home';
import { Demo } from './sections/demo/demo';

export const Main: React.FC = () => {
  return (
    <>
      <Home />
      <Demo />
    </>
  );
};
