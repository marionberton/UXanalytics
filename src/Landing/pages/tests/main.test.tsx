import React from 'react';
import { mount } from 'enzyme';

import { Main } from '../main';

describe('<Main />', () => {
  it('renders the component', () => {
    const component = mount(<Main />);

    expect(component.render()).toMatchSnapshot();
  });
});
