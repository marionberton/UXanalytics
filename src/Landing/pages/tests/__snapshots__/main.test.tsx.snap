// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> renders the component 1`] = `
Array [
  <section
    class="mt-24"
  >
    <div
      class="container"
    >
      <div
        class="flex lg:items-center lg:flex-row flex-col"
      >
        <div
          class="lg:w-1/2 lg:order-first order-last text-center lg:text-left mt-10 md:m-0"
        >
          <h1
            class="text-5xl text-darkgrey font-title font-semibold leading-none"
          >
            Stop copying, and
            <br />
             start doing competitor 
            <br />
             UX analysis
          </h1>
          <p
            class="mt-6 text-darkgrey"
          >
            Create excellent user experience, by analyzing
            <br />
            design metrics of your product with your direct
            <br />
            competitors websites.
          </p>
          <button
            class="bg-btnred text-white font-title font-semibold py-2 px-4 rounded-full lg:w-1/3 mt-10"
            type="button"
          >
            Try Demo Now
          </button>
        </div>
        <div
          class="lg:w-1/2 mb-12"
        >
          <img
            alt=""
            src="home.png"
          />
        </div>
      </div>
    </div>
  </section>,
  <section
    class=" mt-32"
  >
    <div
      class="container"
    >
      <div
        class="flex justify-center content-center items-center text-center flex-col"
      >
        <div>
          <img
            alt=""
            class="h-auto lg:w-3/4 mx-auto"
            src="demo.jpg"
          />
        </div>
        <div>
          <h1
            class="text-4xl text-darkgrey font-title font-semibold leading-none"
          >
            Who is UX Analytic for?
          </h1>
          <p
            class="mt-6 text-darkgrey text-xl"
          >
            Our analysis service empower designer to make better design decisions. 
            <br />
             We analized over 250 websites and app for you.
          </p>
        </div>
      </div>
    </div>
  </section>,
  <section
    class="mt-24 py-20 bg-lightmauve benefit"
  >
    <div
      class="container"
    >
      <div
        class="flex sm:justify-between items-center sm:flex-row flex-col"
      >
        <div
          class="p-2 order-first "
        >
          <h1
            class="text-lg md:text-xl lg:text-2xl xl:text-3xl text-darkgrey font-title"
          >
            Actionable insight 
            <span
              aria-label="lightbulb"
              role="img"
            >
              ðŸ’¡
            </span>
          </h1>
          <p
            class="text-xl md:text-2xl lg:text-3xl xl:text-4xl text-darkgrey font-title font-bold"
          >
            Focus on best practices 
            <br />
            for your design
          </p>
        </div>
        <div
          class="flex flex-col box-content sm:w-1/2"
        >
          <div
            class="flex flex-row rounded-md bg-white py-5 pnpm x-10 mb-5 sm:items-center"
          >
            <div>
              <img
                alt=""
                src="topicon.svg"
              />
            </div>
            <div
              class="ml-5"
            >
              <h1
                class="text-darkgrey text-base sm:text-lg md:text-xl font-bold"
              >
                Insights
              </h1>
              <p
                class="text-parablack"
              >
                Find out which design patterns work best in your industry.
              </p>
            </div>
          </div>
          <div
            class="flex flex-row rounded-md bg-white py-5 px-10 sm:items-center"
          >
            <div>
              <img
                alt=""
                src="bottomicon.svg"
              />
            </div>
            <div
              class="ml-5"
            >
              <h1
                class="text-darkgrey text-base sm:text-lg md:text-xl font-bold"
              >
                Upskilling
              </h1>
              <p
                class="text-parablack"
              >
                Add quantitative, metrics to your design and boost the experience for your users.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    class="mt-24"
  >
    <div
      class="container"
    >
      <div
        class="flex justify-around content-start items-center lg:text-left text-center lg:flex-row flex-col"
      >
        <div
          class="mb-12 lg:m-0"
        >
          <img
            alt=""
            src="mansearching.svg"
          />
        </div>
        <div>
          <h1
            class="text-4xl text-darkgrey font-title font-semibold"
          >
            See it in action
          </h1>
          <p
            class="text-4xl text-mauve font-title font-semibold"
          >
            Try UX Analytic now, and get early access for the beta release.
          </p>
          <button
            class="bg-btnred text-white font-title font-semibold py-2 px-4 rounded-full mt-10"
            type="button"
          >
            Try Demo Now
          </button>
        </div>
      </div>
    </div>
  </section>,
]
`;
